leftwm-core/src/handlers/command_handler.rs

```diff
@@ -269,6 +270,7 @@ fn toggle_state(state: &mut State, window_state: WindowState) -> Option<bool> {
     state.handle_window_focus(&handle);
     match window_state {
         WindowState::Fullscreen => Some(true),
+        WindowState::Maximized => Some(true),
         _ => Some(false),
```

Comment:
```suggestion
        WindowState::Fullscreen | WindowState::Maximized => Some(true),
        _ => Some(false),
```
to make clippy happy